src/content/docs/game-design/godot/2dPlatformerGame.mdx

```diff
@@ -0,0 +1,231 @@
+---
+title: 2D Platformer
+description: This page is a step by step guide to making a 2D platformer
+sidebar:
+  order: 2
+---
+
+This is a tutorial for a basic 2D platformer in Godot. We will be starting with a new project.
+
+Please download the asset pack linked [here](https://brackeysgames.itch.io/brackeys-platformer-bundle) (it is free so you don't need to pay).
+
+## Making the Player
+
+Firstly, click "other node" and add a CharacterBody2D. Rename this to Player. In the inspector, click on "collision" in the CollisionObject2D section and change the layer to only have 2 selected. Now, give your CharacterBody2D 2 child nodes, an AnimatedSprite2D and a CollisionShape2D. In the project menu at the top left, open the project settings, go to the rendering heading in the general tab, click on texture, and change the default texture filter to Nearest.
+
+![Screenshot showing the settings menu](/src/assets/godot/2DPlatformer/screenshot1.png)
+
+Now click on AnimatedSprite2D and click on "animation" in the inspector. Click on Sprite Frames and New SpriteFrames. Now click on the SpriteFrames you just made, this should open a SpriteFrames dock at the bottom of the screen. Here we are going to add the sprite animations for our character. Click on the new animation button at the top left of the new dock and create 4 new animations so you have 5 total. 
```

Comment:
These should be converted to steps (https://wiki.tuhuratech.org.nz/python/flask/forms/#installing-flask-wtf) - this applies to most step by step parts